changelog/README.md

```diff
@@ -0,0 +1,58 @@
+# Multi-file running changelog
+
+To avoid merge conflicts in the CHANGELOG.md file we accepted the multi-file running changelog strategy.
+
+To follow this strategy you should create a `.md` file for every PR. Choose a directory:
+
+- `changelog/unreleased/features` for **Features** changes
+- `changelog/unreleased/bugfixes` for **Bug fixes and improvements** changes
+- `changelog/unreleased/issues` for **Known issues :warning:** changes
+- `changelog/unreleased` for other changes
+
+You can use anything that allow .md format in changelog files.
+
+If you have implemented some features or bugfixes you should describe all of them:
+
+```
+- Description of changes in md format
+- Description of changes in md format also
+```
+
+You can choose any name for your changelog files because the GitHub action will rename files in
```

Comment:
I think it's worth mentioning that you can't create 2 files in 1 directory in 1 PR. I don't think we someone would want that (I don't see any cases for that) but still we should note that.

Reply:
but I mentioned it [here](https://github.com/mapbox/mapbox-navigation-android/pull/6740/files/0df6090a3f31bb3559f877434447707d953af319#diff-861d0d6527afefde8a5f11cd0baad47e651931f7424b83462f0f66875afde016R14)